v1.1/licensing.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Licensing
```

Comment:
I think it might be helpful to title this page `Enterprise Licensing`. Same goes for the filename and the entry in the sidenav.

Reply:
Fixed.